### PR TITLE
Added homepage

### PR DESCRIPTION
**Pull request by Shrinkhlarajpoot**
Added Landing Page

### CODE REVIEW

**Review by palash23jain — APPROVED** [review]
i see you have used id selector in one of the case. Hope you have good enough reason for the same. Looks good to go

File: index.css [hunk 1]
```diff
@@ -0,0 +1,315 @@
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Montserrat:ital,wght@0,400;0,500;0,900;1,600;1,900&family=Recursive:wght@300&family=Roboto+Slab&family=Source+Sans+Pro&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Montserrat:ital,wght@0,400;0,500;0,900;1,600;1,900&family=Recursive:wght@300&family=Roboto+Slab&family=Source+Sans+Pro&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Montserrat:ital,wght@0,400;0,500;0,900;1,600;1,900&family=Recursive:wght@300&family=Roboto+Slab&family=Source+Sans+Pro:ital@0;1&display=swap');
+
```
Comment (msdeshmukh009) [inline index.css:4]: Adding these links in the HTML head will give you a little bit of performance benefit.

File: index.css [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+    margin-top:3rem;
+   width:fit-content;
+   padding:1rem;
+   font-weight: bolder;
+   border-radius: 3px;
+  
+}
+
+
+
```
Comment (msdeshmukh009) [inline index.css:137]: Extra space here.

**Review by sanathkumar26700 — APPROVED** [review]

File: index.css [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+ 
+ .rights{
+     width:100%;
+     height:5rem;
+     text-align: center;
+    color:var(--text--color);
+    padding-top:1rem;
+    font-size: 1.2rem;
+
+
```
Comment (msdeshmukh009) [inline index.css:237]: Extra space here as well

**Review by msdeshmukh009 — APPROVED** [review]
The rest is looking good.

File: index.css (hunk 1 above)
Comment (Shrinkhlarajpoot) [reply index.css:4]: Okay i will look into it

**Pull request merged**